rs-libc/build.rs

```diff
@@ -42,6 +42,7 @@ fn main() {
         "librsc.a"
     };
 
+
     build
         .define(
             "weak_alias(old,new)",
```

Comment:
-ffreestanding is not supported by MSVC, I am unable to find the correct substitute.

Reply:
Take a look at https://wiki.osdev.org/Visual_Studio#Compiler_Options and see what would be good. At least `/NODEFAULTLIB` I'd think

Reply:
`cc` doesn't pass the flag `/NODEFAULTLIB` to the `link.exe`